install_sct

```diff
@@ -674,7 +674,7 @@ else
   print info "Installing data..."
   run rm -rf "$SCT_DIR/$DATA_DIR"
   run mkdir -p "$SCT_DIR/$DATA_DIR"
-  for data in PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models; do
+  for data in PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models deepreg_models; do
```

Comment:
I don't have a strong opinion on this but I think that your suggestion of "automatically download the models at runtime" may be interesting to not increase the installation size of SCT with models that may be never used by the users. 

> Doing this might cause issues for user-run batch scripts, though, since they might simultaneously download copies of the same files in parallel

Oh yes, in this case it can be problematic but I would agree that it is probably a minimal concern. But then again, I don't know much about it ^^'